src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+import Articles from "./components/Articles";
+import Topics from "./components/Topics";
```

Comment:
You're not using the Topics component yet. Either remove or comment out this line for now.